app/application.go

```diff
@@ -169,6 +169,11 @@ func (app *App) setupState(stateBytes []byte) error {
 	}
 	app.Context.feePool.SetupOpt(&initial.Governance.FeeOption)
 
+	err = app.Context.govern.WithHeight(app.header.Height).SetNetworkDelegOptions(initial.Governance.DelegOptions)
```

Comment:
I think we should use ,hard coded constants to set options for now . Until we decide to add things to genesis